html/banner_section.html

```diff
@@ -0,0 +1,47 @@
+<section>
+  <span id='top_anchor' class='anchors'></span>
+  <div class="slider banner_slick">
+    <div>
+      <div class='banner_inside_slide'>
+        <img src='img/banner_img11.jpg' alt='#' />
+        <div>
+          <p class='banner_heading'>Improv Shows</p>

```

Comment:
lets change this to h2 then you don't have to use so many classes and it is semantic